### DISCUSSION TITLE
Mobile on-screen buttons don't work, plus suggestions

### DISCUSSION THREAD

**Discussion by rjd1922** [ayvacs/gba.js.org#19] [Ideas]
I can't play this on Android Chrome as the on-screen buttons do nothing. Also, do you have any plans to support the real-time clock for Pokémon Ruby, Sapphire, and Emerald? It would also be nice to unlock all the World-e levels in Super Mario Advance 4.

**Comment by rjd1922** [comment]
Also, add Metroid: Zero Mission and Mario vs. Donkey Kong (with the e-Reader levels unlocked if possible).

**Comment by ayvacs** [comment]
done
